server/api/store.go

```diff
@@ -720,3 +761,12 @@ func parseStoreLimitType(typeName string) (storelimit.Type, error) {
 	}
 	return typeValue, err
 }
+
+func checkProgressAction(action string) error {
+	switch action {
+	case metapb.NodeState_Preparing.String(), metapb.NodeState_Removing.String():
+		return nil
+	default:
```

Comment:
I think the user does not need to specify an `action`. one store must has one action?

Reply:
For those stores which don't in the preparing or removing state, they don't have a progress indicator.

Reply:
we can return empty if there don't have a progress indicator.